ansible/roles/csm.ncn.pdsh/defaults/main.yml

```diff
@@ -0,0 +1,33 @@
+#
+# MIT License
+#
+# (C) Copyright 2021-2024 Hewlett Packard Enterprise Development LP
```

Comment:
This file is new, it and all the other new files should just have a copyright year of 2024.